docs/guide/features.md

```diff
@@ -368,6 +370,24 @@ By default, the worker script will be emitted as a separate chunk in the product
 import MyWorker from './worker?worker&inline'
 ```
 
+### Import with Constructors
```

Comment:
Shall we put this section before `Import with Query Suffixes` since it said this is the recommended way to create workers so readers would feel the docs emphasizes this than suffixes?

Reply:
I think this makes sense, but I didn't suggest it earlier because there's no way to do `?inline` with constructors yet (if i'm not mistaken). But I'm fine either way too.

Reply:
I think the end goal is to remove query suffixes everywhere for Vite, but we can't put `inline` in worker options anyways since that's non-standard. I'm actually not sure what's the right call here but it seems like it needs more discussion.

Reply:
yep, this is compile-time option don't let it into runtime option. 😊 so how about 
```ts
const worker = new Worker(new URL(/* #__INLINE__ */'./worker.js', import.meta.url))
```

Reply:
I think we would want to refrain from special comments too 😬 IIRC Rollup has a history of avoiding that too (e.g. `webpackChunkName`) and only supported pure comments thus far. I think Vite should follow suit too and find some alternatives. I'm not sure what yet, but maybe we can create a new issue for it to toss some ideas, and have it discussed soon.

Reply:
ok, https://github.com/vitejs/vite/issues/7352 let us discuss this.